LCLSGeneral/LCLSGeneral/POUs/Functions/FB_EcatDiag.TcPOU

```diff
@@ -65,14 +77,31 @@ FOR nIterator := 1 TO (UINT_TO_INT(fbGetAllSlaveStates.nSlaves) ) BY 1
 	IF NOT( (astTermStates[nIterator].deviceState = EC_DEVICE_STATE_OP) AND (astTermStates[nIterator].linkState = EC_LINK_STATE_OK)) THEN
 		q_xAllSlaveStatesGood := FALSE;
 	END_IF
+    IF q_anTermStates[nIterator] <> astTermStates[nIterator].deviceState THEN
+        // Log message
+        fbJson.StartObject();
+        fbJson.AddKey('ecat_');
```

Comment:
This a WIP?

Reply:
Noo.... yes. I'll remove this stuff/ revert this block. The ecat diags I just added are more thorough.